packages/modules/imodel-browser/rollup.config.js

```diff
@@ -13,12 +13,9 @@ import * as packageJson from "./package.json";
 
 const rollupConfig = {
   input: "src/index.ts",
-  external: [
-    /@itwin\/itwinui-react(\/.*)?/,
-    /classnames/,
-    /@bentley\/icons-generic-webfont/,
-    /react-intersection-observer/,
-  ],
+  external: Object.keys(packageJson.dependencies).map(
+    (dep) => new RegExp(`${dep}(/.*)?`, "g")
```

Comment:
do you need the "g" here?

![image](https://user-images.githubusercontent.com/11051042/129081109-2b9a2a28-4be3-4dc3-bd00-e4c1dcd98651.png)

Reply:
well, doesnt change much, g is the regexp switch for recursive search, I mainly used what Vilius had in manage-versions